android/build.gradle

```diff
@@ -2,21 +2,21 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "33.0.0"
-        minSdkVersion = 24
-        compileSdkVersion = 33
-        targetSdkVersion = 33
-        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
-        ndkVersion = "23.1.7779620"
-        kotlinVersion = "1.6.0"
+        buildToolsVersion = "34.0.0"
+        minSdkVersion = 23
```

Comment:
Lets keep min sdk 24 thats what it was at before. We could even bump it up more if we wanted? Helium Mobile is at 30